graphite.lua

```diff
@@ -462,4 +463,36 @@ _M.status = function()
 	return status
 end
 
+local function pack(...)
+	return {r = {...}, n = select('#', ...)}
+end
+
+local function wrap_expirationd_func(func, name, callback, args)
+	return function (...)
+		local start = clock.time64()
+		local ret = pack(func(...))
+		callback(name, start, clock.time64(), args)
+		return unpack(ret.r, 1, ret.n)
+	end
+end
+
+_M.add_expirationd_callbacks = function(is_tuple_expired_callback,
+		process_expired_tuple_callback, args)
+	local expirationd = require('expirationd')
```

Comment:
imho, it's better to assert on init stage if there is anything wrong with expirationd module